src/digital/v2_compat.rs

```diff
@@ -115,6 +117,7 @@ mod tests {
         }
     }
 
+    #[cfg(feature = "unproven")]
```

Comment:
You added this.

Reply:
`v1::StatefulOutputPin` is `unproven`, so this test wouldn't build without the flag anyway.